ios/FluentUI/People Picker/ContactCollectionViewLayout.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class ContactCollectionViewLayout: UICollectionViewFlowLayout {
+    @objc weak var delegate: UICollectionViewDelegateFlowLayout?
```

Comment:
I'm very confused by this line.
A UICollectionViewDelegateFlowLayout is supposed to a UICollectionViewDelegate, which is supposed to be set on a UICollectionView. In this class, you are declaring that the layout (UICollectionViewFlowLayout) needs a UICollectionViewDelegateFlowLayout and setting the layout as its own delegate. This seems like an anti-pattern.

If you expose a delegate property, it's with the intention to have some other object be that delegate. Hence the term delegate, since you are delegating tasks to some other entity.

I think I understand the intent here.
You wanted the functionality from some of the methods exposed in the UICollectionViewDelegateFlowLayout protocol to use in this class.
Here's how you should fix this up:

- Get rid of this delegate property.
- Get rid of extension ContactCollectionViewLayout: UICollectionViewDelegateFlowLayout
- Move the methods from that extension inside the ContactCollectionViewLayout class and rearrange the parameters appropriately.